nemo/lightning/fabric/plugins.py

```diff
@@ -1,19 +1,19 @@
 from contextlib import contextmanager
-from typing import Any, Generator, Literal, TypeVar, TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Generator, Literal, TypeVar
 
 import torch
 from lightning_fabric.plugins.precision import MixedPrecision
 from torch import nn
 from torch.optim import Optimizer
 
-from nemo.utils.import_utils import safe_import
 from nemo.lightning.fabric.conversion import to_fabric
 from nemo.lightning.pytorch.plugins.mixed_precision import (
-    MegatronMixedPrecision, 
     DtypeConfig,
+    MegatronMixedPrecision,
     get_optim_config,
-    update_config_with_dtype_overrides
+    update_config_with_dtype_overrides,
 )
+from nemo.utils.import_utils import safe_import
```

Comment:
is this used?

Reply:
Yeah, [here](https://github.com/NVIDIA/NeMo/blob/c1cfa9c0cee7a674e804401a90d2bf4a40afd113/nemo/lightning/fabric/plugins.py#L54).